.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  publish:
+    name: Publish to npm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: auto-dist-tag
+        run: npx auto-dist-tag@1 --write
```

Comment:
Out of curiosity, any reason we need `@1`? Was comparing this with the setup added in `ember-cli-babel`, which just has `npx auto-dist-tag --write`.